.nuspec/Microsoft.Maui.Controls.targets

```diff
@@ -120,6 +120,7 @@
 		Condition=" '$(DesignTimeBuild)' != 'True' AND '@(MauiXaml)' != ''">
 	    <PropertyGroup>
 		<_MauiXamlCValidateOnly>$(MauiXamlCValidateOnly)</_MauiXamlCValidateOnly>
+		<_MauiXamlCValidateOnly Condition="'$(Configuration)' != 'Release' AND '$(_MauiForceXamlCForDebug)' != 'True'">True</_MauiXamlCValidateOnly>
```

Comment:
Just an FYI a user can name their `$(Configuration)` anything. So like this wouldn't work for an `AppStore` configuration.

It might be better to check other properties for this like `$(Optimize)` or `$(DebugSymbols)`.

You wouldn't need to check `$(EmbedAssembliesIntoApk)`, but this is what Xamarin.Android has used for a long time:

https://github.com/xamarin/xamarin-android/blob/4ed6a222ed07551734b28525dd269d50ac107ea8/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets#L302-L323